django/contrib/auth/hashers.py

```diff
@@ -72,6 +72,10 @@ def make_password(password, salt=None, hasher='default'):
     """
     if password is None:
         return UNUSABLE_PASSWORD_PREFIX + get_random_string(UNUSABLE_PASSWORD_SUFFIX_LENGTH)
+    if not isinstance(password, (bytes, str)):
```

Comment:
Do any tests fail if bytes is removed here? If not, should add a test for that (probably in a separate commit).